progression/src/main/java/me/mykindos/betterpvp/progression/progression/perks/DropMultiplier.java

```diff
@@ -0,0 +1,20 @@
+package me.mykindos.betterpvp.progression.progression.perks;
+
+import me.mykindos.betterpvp.core.config.Config;
+import me.mykindos.betterpvp.core.utilities.UtilMath;
+import net.kyori.adventure.text.Component;
+import org.bukkit.Bukkit;
+
+import java.util.Random;
+
+public interface DropMultiplier {
+
+    default int getExtraDrops(double chance) {
+        int multiplier = (int) Math.floor(chance/100);
+        double newChance = chance % 100;
```

Comment:
There isn't a need to get `chance % 100` since that'll only mess with chances above or equal to 100 since your chance is already on a 1-100 scale.

Reply:
Chance is intended to be able to go over 100.

Reply:
If your chance is, for example, 150, then it's effectively only 50 because 150 % 100 is 50.

Reply:
Chance over 100 is intended to guarantee an extra drop, with the remainder being used to roll for another drop. A chance of 450 is intended to give 4 guaranteed additional drops, and 50% of a fifth.